bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/customizecards.dat

```diff
@@ -0,0 +1,7 @@
+set param_card bsm 1 0.100000e+00
+set param_card bsm 2 0.000000e+00
+set param_card bsm 3 0.100000e+00
+set param_card bsm 4 0.000000e+00
+set param_card bsm 5 0.100000e+00
+set param_card bsm 6 0.000000e+00
+set param_card DECAY 8000001 1.000000e-03
```

Comment:
You will have the same 1MeV width for all the different tprime masses?

Reply:
Hi Sihyun, we just want a narrow decay, so I think this is ok for simplicity's sake